### PR TITLE
UKPE-1235 As a Dev I want to see if endpoints have not been monitored for a longer time

### PR DESCRIPTION
**Pull request by Suremaker**

### CODE REVIEW

File: HealthMonitoring.Management.Core/Configuration/AppSettingsMonitorSettings.cs [hunk 1]
```diff
@@ -20,6 +21,7 @@ public AppSettingsMonitorSettings()
             HealthyResponseTimeLimit = GetTimeSpanOrDefault("Monitor.HealthyResponseTimeLimit", TimeSpan.FromSeconds(3));
             ShortTimeOut = GetTimeSpanOrDefault("Monitor.ShortTimeOut", TimeSpan.FromSeconds(4));
             FailureTimeOut = GetTimeSpanOrDefault("Monitor.FailureTimeOut", TimeSpan.FromSeconds(20));
+            HealthUpdateInactivityTimeLimit = GetTimeSpanOrDefault("Monitor.HealthUpdateInactivityTimeLimit", TimeSpan.FromSeconds(20));
```
Comment (williamlord) [inline HealthMonitoring.Management.Core/Configuration/AppSettingsMonitorSettings.cs]: Isn't this a low default?

File: HealthMonitoring.Management.Core/Endpoint.cs [hunk 2]
```diff
@@ -27,7 +30,7 @@ public void Dispose()
 
         private void UpdateLastModifiedTime()
         {
-            LastModifiedTime = DateTimeOffset.UtcNow;
+            LastModifiedTimeUtc = _timeCoordinator.UtcNow;
```
Comment (williamlord) [inline HealthMonitoring.Management.Core/Endpoint.cs]: Could be part of property?

File: make/Environment.ps1 [hunk 3]
```diff
@@ -1 +1 @@
-$VERSION = "3.2.1.0"
\ No newline at end of file
+$VERSION = "3.2.2.0"
```
Comment (williamlord) [inline make/Environment.ps1]: 3.3.0.0

File: HealthMonitoring.Management.Core/Configuration/AppSettingsMonitorSettings.cs (hunk 1 above)
Comment (Suremaker) [inline HealthMonitoring.Management.Core/Configuration/AppSettingsMonitorSettings.cs]: changed to 1 min

File: make/Environment.ps1 (hunk 3 above)
Comment (Suremaker) [inline make/Environment.ps1]: Fixed

**Pull request merged**